cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-intro.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-intro"]
+= Tutorial: Deploying an application
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-intro
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2023-12-14
+
+== Introduction
+
+After successfully provisioning your cluster, you can deploy an application on it. This application allows you to become more familiar with some of the features of {product-title} (ROSA) and Kubernetes.
+
+=== Lab overview
+In this lab, you will complete the following set of tasks designed to help you understand the concepts of deploying and operating container-based applications:
+
+* Deploy a Node.js based app by using S2I and Kubernetes Deployment objects.
+* Set up a continuous delivery (CD) pipeline to automatically push source code changes.
+* Explore logging.
+* Experience self healing of applications.
+* Explore configuration management through configmaps, secrets, and environment variables.
+* Use persistent storage to share data across pod restarts.
+* Explore networking within Kubernetes and applications.
+* Familiarize yourself with ROSA and Kubernetes functionality.
+* Automatically scale pods based on loads from the Horizontal Pod Autoscaler.
+* Use AWS Controllers for Kubernetes (ACK) to deploy and use an S3 bucket.
+
+You will conduct this lab with either the ROSA CLI or the ROSA web user interface (UI). 
```

Comment:
I keep bouncing this sentence in my head, and I cannot tell why I don't like it...

Maybe "This workshop lab module (or whatever) uses either the ROSA CLI or the {cluster-manager-url} ROSA UI"?

Reply:
Yeah, I definitely had issues with this one as well... I will mull it over a bit before we merge this.